core/domain/feedback_jobs_one_off_test.py

```diff
@@ -94,7 +94,7 @@ def test_message_count(self):
 
         self._run_one_off_job()
 
-        thread_summaries = feedback_services.get_thread_summaries(
+        thread_summaries, _ = feedback_services.get_thread_summaries(
```

Comment:
Well, you might as well test the second arg... (probably in feedback_services_test, though, I guess).

In general please add meaningful tests for all new functionality. I _think_ I've commented on this in several PRs now, so in the future, this should go without saying. You want your code to not break in the future!